meshtastic/interfaces/ble/lifecycle_ownership_runtime.py

```
@@ -0,0 +1,939 @@
+        bool
+            First authoritative bool probe result; otherwise ``False``.
+        """
+        if owner is None:
+            return False
+        for member_name in member_names:
+            member = getattr(owner, member_name, None)
+            if callable(member) and not _is_unconfigured_mock_callable(member):
+                try:
+                    result = member()
+                except Exception:  # noqa: BLE001 - probe remains best effort
+                    continue
```

Ruff (0.15.6)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)